docs/openapi/paths/professors/remind_{id}.yaml

```diff
@@ -0,0 +1,20 @@
+post:
```

Comment:
Then this file would be renamed as professors_{id}_remind.yaml

Reply:
done.